js/core/tv.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * TV Simulator '99
+ * @Author: zshall 
+ * @Date: 2017-05-02 22:34:58 
+ * @Last Modified by: zshall
+ * @Last Modified time: 2017-05-03 00:31:21
+ */
+const maxWidth = 1200;
+const maxHeight = 1000;
+const warmupTime = 3000;
+var firstStart = true;
+const classes = {
+    Channel12
```

Comment:
What I'd like to do is do it dynamically from the guide XML, but I'm stuck trying to figure out how to do that without using `exec`. I'd like to call `new Channel${number}();`.

My first idea is to follow this advice: http://stackoverflow.com/a/9804142/970180 - use a namespace that all channels would exist in, and create them using bracket notation.